src/components/OrderSidebar.js

```diff
@@ -0,0 +1,104 @@
+import React from 'react';
+import { useSelector, useDispatch } from 'react-redux';
+import {
+  List,
+  ListItem,
+  ListItemText,
+  Button,
+  Typography,
+  Box,
+} from '@mui/material';
+import { useNavigate } from 'react-router-dom';
+import { clearCart } from '../redux/shoppingCart';
+
+const OrderSidebar = () => {
+  //path to env variable
+  const envPath = process.env.REACT_APP_API_BASE_PATH;
+
+  const navigate = useNavigate();
+  const dispatch = useDispatch();
+
+  // values come from Redux store
+  const itemsInCart = useSelector((state) => state.cart);
+  const subtotal = useSelector((state) => state.cart.subtotal); //total price of all items
+  const tax = useSelector((state) => state.cart.tax);
```

Comment:
The tax in the cart is the percentage (eg: 0.15 for 15%). But right now we're displaying it as a dollar amount:

![](https://github.com/Code-the-Dream-School/dd-prac-team4-front/assets/9094098/dc3bd9f2-1d9c-4b13-b51d-aef37d663ddc)

If we want to display the dollar amount of how much will be added to the subtotal due to taxes, then we should multiply it by `subtotal`:

```suggestion
  const tax = useSelector((state) => state.cart.tax) * subtotal;
```